From 1d69af0a75bbb48e5c541c8bd82590b2332f4726 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Fri, 24 Sep 2021 08:34:55 +0530
Subject: [PATCH] Remove deprecated SchedulingPolicyService

SchedulingPolicyService hal is deprecated in
Android 12, so removing it from device compatablity matrix.

Tracked-On: OAM-99303
Signed-off-by: svenate <salini.venate@intel.com>
---
 groups/device-specific/caas/compatibility_matrix.xml | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index 6fe200e..336b344 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -7,14 +7,6 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl" optional="false">
-        <name>android.frameworks.schedulerservice</name>
-        <version>1.0</version>
-        <interface>
-            <name>ISchedulingPolicyService</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <hal format="hidl" optional="false">
         <name>android.frameworks.sensorservice</name>
         <version>1.0</version>
-- 
2.17.1

